bigframes/core/compile/scalar_op_compiler.py

```diff
@@ -1070,8 +1070,16 @@ def floordiv_op(
     )
 
 
-def _is_float(x: ibis_types.Value):
-    return isinstance(x, (ibis_types.FloatingColumn, ibis_types.FloatingScalar))
+def _is_bignumeric(x: ibis_types.Value):
+    if not isinstance(x, ibis_types.DecimalValue):
+        return False
+    # Should be exactly 76 for bignumeric
+    return x.precision > 70
```

Comment:
Can you update it into `== 76` as you handle the numeric casting at line 1102 in this line?

Reply:
I care about the pre-cast type information, and I don't trust ibis to have the precision/scale numbers right.